ansible_collection/hpe/nimble/galaxy.yml

```diff
@@ -0,0 +1,12 @@
+namespace: "hpe"
+name: "nimble"
+version: "1.0.0"
+authors:
+    - HPE Nimble Storage Ansible Team (@ranjanal) <hpe_nimble-ansible-team@nimblestorage.com>
```

Comment:
Does that email actually exist?

Reply:
have changed to nimble-dcs-storage-automation-eng@hpe.com